src/main/java/com/google/devtools/build/lib/bazel/commands/ModCommand.java

```diff
@@ -194,6 +193,7 @@ private BlazeCommandResult execInternal(CommandEnvironment env, OptionsParsingRe
         EvaluationContext.newBuilder()
             .setParallelism(threadsOption.threads)
             .setEventHandler(env.getReporter())
+            .setKeepGoing(options.getOptions(KeepGoingOption.class).keepGoing)
```

Comment:
I wonder if we should just set this to true unconditionally. After all, it's been the case so far, and IMO for an informational subcommand like `bazel mod`, most people would want to set `--keep_going` as a default.

Reply:
While it's technically been enabled so far, I don't think that it had any effect in practice since the relevant `SkyFunction`s are not catching errors. But I still agree that `bazel mod` commands should probably keep going by default. I've also implemented this for `bazel mod tidy`.